meetings/2023-02-01.md

```diff
@@ -50,36 +57,125 @@ URL: https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-fe
 * SC: Some considerations and discussions:
   * https://gitter.im/solid/specification?at=63d817920c94855213d77fb4 and after.
 * SC: Any objections to stick to Gitter->Matrix transition?
+* SC: I don't know some details: what happens to URLs, does it redirect.. what happens to current admins of channels. If anyone knows anything about these considerations, please share so we can be prepared.
+* SC: I posted on the channel about why sticking to this change is simpler. If there are other approaches, please share.
+* WT: I support that, sticking with that plan. 
+* PAC: Just to mention I have never used Gitter, other than via a Matrix client, works fine.  
+* SC: W3C also have Matrix available but doesn't seem active. some working groups made that move long ago, but general channels seems nobody is really using. 
+* TBL: Do we have any alternative?
+* SC: Only alternative proposed is Solid Chat but there are questions around that: how ready it is, how much support it will have.. Roadmap to switching is a separate topic.
+* TBL: We need to have a roadmap for using Solid Chat. 
+* TBL: You can get notifications on a webpage (on Solid Chat ???). 
+* SC: Since this change is next week, and moving the CG needs planning, I want to look at the simplest thing right now. If there are any objections to let the switch however it happens to go through. 
+* TBL: We need to switch the logger 
+* SC: That's not a dependency of CG. That's Solid Chat's TODOs. 
+* TBL: For the community to move to Solid Chat. 
+* SC: To wrap up: on Feb 6 Gitter is going to do something and switch to Matrix. Can we just roll with that and use Matrix? Some people are already using it. 
+* TBL: If you go to gitter will you get a Matrix client? 
+* SC: That's going to route you using the Element app. 
+* TBL: Will Gitter be a Matrix node? 
+* SC: I think it is already because people are able to see.. at least there's a bridge. I assume they will preserve existing channels. 
+* WT: Yes, it will remain as a Matrix node so if you go to gitter.im you will arrive at the same node. 
+* SC: What happens to existing roles/accounts? I have an account on Matrix and I'm an admin but I don't have admin permissions to the bridge node on Matrix. 
```

Comment:
```suggestion
* SC: I think it is already, because people are able to see. At least there's a bridge. I assume they will preserve existing channels. 
* WT: Yes, it will remain as a Matrix node, so if you go to gitter.im, you will arrive at the same node. 
* SC: What happens to existing roles/accounts? I have an account on Matrix, and I'm an admin, but I don't have admin permissions to the bridge node on Matrix. 
```